App/Syncthing/syncthing_sync_check.sh

```diff
@@ -0,0 +1,284 @@
+#!/bin/ash
+
+. /mnt/SDCARD/App/Syncthing/syncthingFunctions.sh
+. /mnt/SDCARD/spruce/scripts/helperFunctions.sh
+
+SYNCTHING_CONFIG_DIR="${SYNCTHING_CONFIG_DIR:-/mnt/SDCARD/App/Syncthing/config}"
+API_ENDPOINT="http://localhost:8384/rest"
+CONFIG_XML="$SYNCTHING_CONFIG_DIR/config.xml"
+CHECK_INTERVAL=2  # Interval between checks in seconds
+MAX_API_RETRIES=15
+API_RETRY_INTERVAL=2
```

Comment:
Any reason not to make these intervals 1? They seem like quick checks that take much less than 1 second, but not sure if you found 2 seconds works better for some reason.

Reply:
That's a good point, the first `CHECK_INTERVAL` is set at 2 seconds because in the same function I update the display and doing it every second could be jarring and not give the person time to read.

The `API_RETRY_INTERVAL` isn't tied to displaying any information so I could poll quicker, gracias:
0e0079269c035a3aa88f52753e3204e0b20dc769